### PR TITLE
Update User Guide: Add new task & delete existing task

### LINKED ISSUE #37: Update User Guide

### PR DESCRIPTION
**Pull request by hanweic53**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -122,19 +127,19 @@ Examples:
...
 
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
+* Deletes the task at the specified `INDEX`.
+* The index refers to a valid index number shown in the displayed task list.
 * The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd task in tr4cker.
-* `find Betsy` followed by `delete 1` deletes the 1st task in the results of the `find` command.
```
Comment (cwenling) [inline docs/UserGuide.md:137]: this function works in Tr4cker too. maybe we should keep the corresponding version of this line?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -67,19 +67,24 @@ Shows a message explaning how to access the help page.
...
-A person can have any number of tags (including 0)
+A task can have any number of tags (including 0)
 </div>
 
 Examples:
-* `add n/task 1 dl/2020-09-08 des/task 1 description`
-* `add n/task 2 dl/2020-08-09 2359 des/task 2 description t/urgent`
+* `add n/CS1101S Quiz dl/2020-01-25 des/Post-Lecture quiz`
+* `add n/CS1231S Homework Assignment dl/2020-09-08 2200 des/task 1 description t/assignment t/urgent`
+
```
Comment (cwenling) [inline docs/UserGuide.md:87]: this might be an extra line

File: docs/UserGuide.md (hunk 1 above)
Comment (hanweic53) [reply docs/UserGuide.md:137]: Yes you're right.

**Review by cwenling — APPROVED** [review]
looks good to merge

**Pull request merged**